settings/uboone/PandoraSettings_MicroBooNE.xml

```diff
@@ -507,14 +507,25 @@
                 <algorithm type = "LArSvmPfoCharacterisation">
                     <TrackPfoListName>TrackParticles3D</TrackPfoListName>
                     <ShowerPfoListName>ShowerParticles3D</ShowerPfoListName>
-                    <SvmFileName>/r05/dune/MachineLearningData/PandoraSvm_PfoCharacterisation_MicroBooNE_mcc7.xml</SvmFileName>
+                    <UseThreeDInformation>true</UseThreeDInformation>
+                    <SvmFileName>/r05/dune/MachineLearningData/PandoraSvm_PfoCharacterisation_MicroBooNE_mcc7_3D.xml</SvmFileName>
```

Comment:
No longer need fully qualified svm data file names; but do need to ensure that file lives in a path provided by the colon-separated FW_SEARCH_PATH environment variable (first file in paths with given name will be used).